action.yml

```diff
@@ -5,18 +5,14 @@ inputs:
     description: Github token
     required: true
     default: ${{ github.token }}
-  lcov-file:
-    description: The location of the lcov.info file
-    required: false
-    default: './coverage/lcov.info'
+  coverage-file-path:
```

Comment:
The input is named `coverage-file` in https://github.com/kcjpop/coverage-comments/pull/6/files#diff-27653c212e1cfe533e4eb2f7d0d3f89604c9de48a09583b4cbbbcbd08a07da79R16